src/Moat/Pretty/Kotlin.hs

```diff
@@ -272,3 +272,8 @@ toUpperFirst = \case
 
 noIndent :: String
 noIndent = ""
+
+maybeParcelableTyVars :: [String] -> [Interface] -> [String]
+maybeParcelableTyVars tyVars ifaces
+  | Parcelable `elem` ifaces = map (++ " : Parcelable") tyVars
+  | otherwise = tyVars
```

Comment:
nit: When I see `maybeX` I usually image it returning `Maybe X`. Maybe `addParcelableToTyVars`?

Reply:
I'm thinking `addTyVarBounds`, since we could theoretically support more than just `Parcelable` here.

Reply:
I like it!